cmd/influxd/inspect/verify_seriesfile.go

```diff
@@ -9,10 +9,96 @@ import (
 	"sort"
 	"sync"
 
+	"github.com/influxdata/influxdb/v2/logger"
 	"github.com/influxdata/influxdb/v2/tsdb"
+	"github.com/spf13/cobra"
 	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
 )
 
+type args struct {
+	dir        string
+	db         string
+	seriesFile string
+	verbose    bool
+	concurrent int
+}
+
+func NewVerifySeriesfileCommand() *cobra.Command {
+	var arguments args
+	cmd := &cobra.Command{
+		Use:   "verify-seriesfile",
+		Short: "Verifies the integrity of series files.",
+		Args:  cobra.NoArgs,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			cmd.SetOut(os.Stdout)
+
+			config := logger.NewConfig()
+			config.Level = zapcore.WarnLevel
+			if arguments.verbose {
+				config.Level = zapcore.InfoLevel
+			}
+			log, err := config.New(os.Stderr)
+			if err != nil {
+				return err
+			}
+
+			v := NewVerify()
+			v.Logger = log
+			v.Concurrent = arguments.concurrent
+
+			var db string
+			if arguments.seriesFile != "" {
+				db = arguments.seriesFile
+			} else if arguments.db != "" {
+				db = filepath.Join(arguments.dir, arguments.db, "_series")
+			}
+			if db != "" {
+				_, err := v.VerifySeriesFile(db)
+				return err
+			}
+
+			dbs, err := ioutil.ReadDir(arguments.dir)
+			if err != nil {
+				return err
+			}
+
+			var hasError bool
+			for _, db := range dbs {
+				if !db.IsDir() {
+					continue
+				}
+				filePath := filepath.Join(arguments.dir, db.Name(), "_series")
+				_, errNew := v.VerifySeriesFile(filePath)
+				if errNew != nil {
+					v.Logger.Error("Failed to verify series file",
+						zap.String("filename", filePath),
+						zap.Error(errNew))
+					hasError = true
+					err = errNew
+				}
+			}
+			if hasError {
+				v.Logger.Info("Some files failed verification, see logs for details")
```

Comment:
```suggestion
				return errors.New("Some files failed verification, see logs for details")
```
And then `return nil` on line 84 for the success case